markdown/chapters/meandeck/basics.md

```diff
@@ -4,74 +4,138 @@ order: 0
+There are three steps that need to occur to enable you to win (at least most of
+the time).  
+These are as follows:
 
 1. Resolve Doomsday
-1. Draw into your pile
-1. Consume enough cards out of the pile so that TO's trigger wins you the game
-
-Often enough, steps 2 and 3 go together by using a *cantrip* effect. This is
-typically a draw spell like Preordain or a free cycle effect like Street Wraith,
-which helps *dig* into the pile. This can be done in a single turn or spread
-over multiple turns, utilising your draw step as a way to *dig* into the pile.
+1. Draw into your pile enough so that the Oracle's trigger can win you the game
+1. Resolve Thassa's Oracle
+
+<row variant="centered">{{!DD}}{{!TO}}</row>
+
+Out these steps, we will start by detailing how to achieve steps 2. and 3. as
+these are the first steps that are unique to the Doomsday archetype. Other decks
+like Omnitell have similar transferable skills in order to resolve a specific
+key spell however it is the *pile* construction that is often the trickiest
+element of playing the deck for newer players of the archetype.
+
+<row variant="centered">{{!Pre}}{{!SW}}</row>
+
+The first consideration to make is how you will *dig* into the pile.  
+We will use the phrase *dig* to mean to initially draw into the five cards you
+have stacked with Doomsday.  
+The most common way to *dig* into your pile is via a *cantrip* effect. This is a
+spell or ability which draws a card upon resolution. Typically this is provided
+by one of the commonly played {U} filter effects like {{Preordain}} or a free
+cycle effect like {{Street Wraith}}. If you are not under pressure you can also
+simply pass the turn to the opposing player and use your next turn's draw step
+in order to *dig* into the *pile*. When this is performed it is simply known as
+a *pass the turn pile* or *PTT pile*.
+
+Once you have decided what method you will *dig* into your pile with you need to
+work out how you are going to get through enough cards that the trigger from
+Thassa's Oracle will win you the game. Sometimes this means having up to two
+cards left in your library on its resolution, other times it means needing to
+have no cards left in your library on resolution. The former is known as an
+*imperfect pile* and the latter a *perfect pile*. An *imperfect pile* is named
+as such because if your opponent is able to remove the Oracle from play prior to
+resolution of its trigger then you will not win the game.
+
+In this section we will explore some of the simplest and most commonly used
+piles in order to achieve a *perfect pile* for your Oracle trigger to resolve.
+
+In each example we shall present any mana requirements or card requirements
+followed by the pile as a visual display.
+
+If we take the following example:
 
-## Single Cantrip + Predict Pile
+> {B}{B}{B} + {U}{U} + Cantrip
 
-> {B}{B}{B} + {1}{U} + Cantrip
+This cost shows we require {B}{B}{B} for casting Doomsday. {U}{U} for casting
```

Comment:
How about a level 2 heading around here that looks like "Pile Notation"?

Reply:
Added